src/uwtools/config/support.py

```diff
@@ -53,6 +55,36 @@ def log_and_error(msg: str) -> Exception:
     return UWConfigError(msg)
 
 
+def represent_namelist(dumper: yaml.Dumper, data: f90nml.Namelist) -> yaml.nodes.MappingNode:
+    """
+    Convert f90 Namelist objects to ordered dicts and serialize.
```

Comment:
```suggestion
    Convert f90nml Namelist to OrderedDict and serialize.
```

Reply:
If I'm not mistaken, this method is called by the YAML library to operate on a single value, in which case plural terminology would not be accurate.